### DISCUSSION TITLE
QNAP Container station

### DISCUSSION THREAD

**Discussion by Krisso82** [wolveix/satisfactory-server#182] [Show and tell]
Hi, for anyone interested here is the command for setting up within QNAP container station.

```shell
docker create --name=saticfactory --net=host --restart=always -e MAXPLAYERS=4 -e PUID=1000 -e PGID=100 -e STEAMBETA=false -e TZ=Australia/Melbourne -v /share/Container_Configs/satisfactory_Config:/config -p 7777:7777 -p 15000:15000 -p 15777:15777 wolveix/satisfactory-server:latest
```

**Comment by wolveix** [comment]
Hey :)

Presumably the only changes here from the stock example on the `README.md` is the inclusion of `--net=host`, and the exact config path? :) 

As a sidenote, `--net=host` removes the need to explicitly define ports. So the command should look like:

```
docker create --name=saticfactory --net=host --restart=always -e MAXPLAYERS=4 -e PUID=1000 -e PGID=100 -e STEAMBETA=false -v /share/Container_Configs/satisfactory_Config:/config wolveix/satisfactory-server:latest
```

**Reply by Krisso82** [reply]
The PGID for my QNAP docker is actually 100 not 1000 when i run the "id dockeruser" command.
But yes the --net=host and exact path are the changes made, aside that was just the command line format for QNAP and thought might be helpful to someone if searching for the QNAP command with not great docker knowledge. :)  That said, i am at a basic level of docker myself.
Please feel free to remove this thread though if it is not a value add to anyone :)

**Reply by wolveix** [reply]
@Krisso82 ah, gotcha! I've updated my initial reply to reflect this :) Thanks for your contribution! I'm sure it'll prove useful to others :D
